### PR TITLE
add unittests

### PR DESCRIPTION
**Pull request by ilyastrodubtsev**

### CODE REVIEW

**Comment by vpetersson** [comment]
@ilyastrodubtsev Please fix the lint issues too.

**Comment by ryanteck** [comment]
I pushed this as a test to our testnet and all looks like it's working fine. :+1:

**Comment by vpetersson** [comment]
> @ilyastrodubtsev Please fix the lint issues too.

Actually. Strike that. We can ignore the lint errors on `diagnostics-program/html_generator.py` for now.

**Comment by vpetersson** [comment]
@ilyastrodubtsev I did notice that these tests are not running as part of GitHub Actions? Could you please add that?

**Review by vpetersson — APPROVED** [review]

File: diagnostics-program/startDiag.sh [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 #Run diag
-rm /opt/nebraDiagnostics/html/index.html
-rm /opt/nebraDiagnostics/html/initFile.txt
-#rm /var/data/public_keys
+rm -f /opt/nebraDiagnostics/html/index.html
+rm -f /opt/nebraDiagnostics/html/initFile.txt
+#rm -f /var/data/public_keys
 #cp /opt/nebraDiagnostics/html/index.html.template /opt/nebraDiagnostics/html/index.html
 
 nginx
```
Comment (vpetersson) [inline diagnostics-program/startDiag.sh:9]: @ilyastrodubtsev I'm pretty sure like 12 will never execute as `nginx` won't fork out. Could you confirm?

File: diagnostics-program/startDiag.sh [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 #sleep 30
-#rm /opt/nebraDiagnostics/html/index.html
+#rm -f /opt/nebraDiagnostics/html/index.html
```
Comment (vpetersson) [inline diagnostics-program/startDiag.sh:11]: Also, please remove all commented out blocks. We can always retrieve them from git if needed.

Comment (ilyastrodubtsev) [reply diagnostics-program/startDiag.sh:11]: I can delete all commented out blocks, but they were there before me.

Comment (vpetersson) [reply diagnostics-program/startDiag.sh:11]: Oh i know. It's just part of the clean up effort.

Comment (ilyastrodubtsev) [reply diagnostics-program/startDiag.sh]: I understand. Done.

File: .github/workflows/python-tests.yml [hunk 3]
```diff
@@ -22,3 +30,8 @@ jobs:
         flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
```
Comment (vpetersson) [inline .github/workflows/python-tests.yml:30]: @ilyastrodubtsev can you change this to run a regular full linting on the code base? If we need to exclude something for now, use the `noqa` annotation.

**Review by vpetersson — APPROVED** [review]
LGMT

**Comment by vpetersson** [comment]
We need to test this in the testnet before deploying to production of course.

**Pull request merged**